bot/cogs/animals.py

```diff
@@ -0,0 +1,122 @@
+            data = data['breeds']
+        if not data:
+            await self.dog(ctx)
+        else:
+            try:
+                fields = [
+                    (
+                        'Weight',
+                        f"{data['weight']['imperial']}lbs.",
+                        True
+                    ),
+                    (
+                        'Height',
+                        f"{data['height']['imperial']}in.",
+                        True
+                    ),
+                    (
+                        'Breed Group',
+                        data['breed_group'],
+                        True
+                    ),
+                    (
+                        'Life Span',
+                        data['life_span'],
+                        True
+                    ),
+                    (
+                        'Bred for',
+                        data['bred_for'],
+                        False
+                    ),
+                    (
+                        'Temperament',
+                        data['temperament'],
+                        False
+                    )
+                ]
+                embed = discord.Embed(title=data['name'])
+                embed.set_image(url=image)
+                for name, value, inline in fields:
+                    embed.add_field(name=name, value=value, inline=inline)
+                await ctx.send(embed=embed)
+            except KeyError:
+                await self.dog(ctx)
+
+    @commands.command(brief='Returns a random cat fact and image')
+    async def cat(self, ctx: commands.context.Context) -> None:
+        """Sets url and gets random cat image and fact."""
+        url = 'https://api.thecatapi.com/v1/images/search'
+        fact_url = 'https://some-random-api.ml/facts/cat'
+
+        await self.animal_fact(ctx, url, fact_url, 'Cat')
+
+    @commands.command(brief='Returns a random bird fact and image')
+    async def bird(self, ctx: commands.context.Context) -> None:
+        """Sets url and gets random bird image and fact."""
+        url = 'https://some-random-api.ml/img/birb'
+        fact_url = 'https://some-random-api.ml/facts/bird'
+
+        await self.animal_fact(ctx, url, fact_url, 'Bird')
+
+    @commands.command(brief='Returns a random fox fact and image')
+    async def fox(self, ctx: commands.context.Context) -> None:
+        """Sets url and gets random fox image and fact."""
+        url = 'https://some-random-api.ml/img/fox'
+        fact_url = 'https://some-random-api.ml/facts/fox'
+
+        await self.animal_fact(ctx, url, fact_url, 'Bird')
+
+    @commands.command(brief='Returns a random panada fact and image')
+    async def panda(self, ctx: commands.context.Context) -> None:
+        """Sets url and gets random panda image and fact."""
+        url = 'https://some-random-api.ml/img/panda'
+        fact_url = 'https://some-random-api.ml/facts/panda'
+
+        await self.animal_fact(ctx, url, fact_url, 'Bird')
```

Comment:
we can add plurals to the aliases, i.e `cats` or `pandas` so that users are more likely to guess the correct command.